db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,33 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
+      t.boolean :login
+      t.string :kind
+      t.string :htpassd_file_path
+      t.text :htpassd_users
+      t.text :ldap_id, :array => true, :default => []
+      t.text :ldap_email, :array => true, :default => []
+      t.text :ldap_name, :array => true, :default => []
+      t.text :ldap_preferred_user_name, array: true, :default => []
+      t.string :ldap_bind_dn
+      t.string :ldap_bind_password
+      t.string :ldap_ca
+      t.boolean :ldap_insecure
+      t.string :ldap_url
+      t.string :request_header_challenge_url
+      t.string :request_header_login_url
+      t.string :request_header_client_ca
+      t.text :request_header_headers, :array => true, :default => []
+      t.string :client_id
+      t.string :client_secret
+      t.string :open_id_sub_claim
+      t.string :open_id_authorization_endpoint
+      t.string :open_id_token_endpoint
```

Comment:
Below is the full example for OpenIDConnect:
```
openshift_master_identity_providers:
- name: openid_connect
  login: true
  challenge: true
  kind: OpenIDIdentityProvider
  clientID: my_client_id
  clientSecret: my_client_secret
  ca: my-openid-ca-bundle.crt
  extraScopes: 
  - email
  - profile
  extraAuthorizeParameters: 
    include_granted_scopes: "true"
  claims:
    id: 
    - custom_id_claim
    - sub
    preferredUsername: 
    - preferred_username
    - email
    name: 
    - nickname
    - given_name
    - name
    email: 
    - custom_email_claim
    - email
  urls:
    authorize: https://myidp.example.com/oauth2/authorize
    token: https://myidp.example.com/oauth2/token
    userInfo: https://myidp.example.com/oauth2/userinfo
```

extraScopes and extraAuthorizeParameters are the ones that @deads2k was referring to.

The ca field and userInfo field also appear to be missing.